iamy.go

```diff
@@ -85,3 +94,37 @@ func init() {
 	}
 	defaultDir = filepath.Clean(dir)
 }
+
+func performVersionChecks() {
+	currentIAMyVersion, _ := semver.Make(Version)
+	log.Printf("current version is %s", currentIAMyVersion)
+
+	fileBytes, err := ioutil.ReadFile(versionFileName)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			log.Printf("could not read version file %s, ignoring: %s", versionFileName, err)
+		}
+		return
+	}
+
+	log.Printf("%s detected", versionFileName)
```

Comment:
```suggestion
	log.Debugf("%s detected", versionFileName)
```

Reply:
we're only using the stdlib `log` package here so `Debugf` isn't a thing. We can use `if *debug {}` checks instead though?

Reply:
hmm, actually we don't have `*debug` either as it's in a separate method 🤔

Reply:
Oh right, you have to pass in the `debug` variable from the calling function and call `debug.Println`